user/yaml/README.md

```diff
@@ -0,0 +1,327 @@
+# Deploy **KubeStellar** service in a cluster

```

Comment:
Add a section on what privileges/assumptions there are when running on k8s:
1. roles for accessing secrets
2. keeping certificates as secrets 

Suggest you add which deployment options were tested.

Reply:
ok